cmd/up.go

```diff
@@ -170,7 +182,52 @@ List of components that kubeapps up installs:
 
 func init() {
 	RootCmd.AddCommand(upCmd)
-	upCmd.Flags().Bool("dry-run", false, "Provides output to be submitted to the server.")
+	upCmd.Flags().Bool("dry-run", false, "Show manifest to be submitted to the k8s cluster without deploying.")
+	upCmd.Flags().StringP("out", "o", "yaml", "Provides manifest format: yaml | json")
+}
+
+func dump(w io.Writer, out string, objs []*unstructured.Unstructured) error {
+	bObjs := [][]byte{}
+
+	switch out {
+	case "json":
+		for _, obj := range objs {
+			j, err := json.MarshalIndent(obj, "", "    ")
+			if err != nil {
+				return fmt.Errorf("can't dump kubeapps manifest: %v", err)
+			}
+			bObjs = append(bObjs, j)
+		}
+
+		b := bytes.Join(bObjs, []byte(fmt.Sprintf(",\n")))
+		// insert '[' to the front
```

Comment:
this isn't the correct format kubectl accepts for multiple json resources (though it's correct json). It should just be json objects separated by a single new line e.g. https://github.com/kubernetes/kubernetes/blob/master/hack/testdata/multi-resource-json.json.

I was able to install the resources with `kubectl` after removing the appending/prepending and just having `b := bytes.Join(bObjs, []byte(fmt.Sprintf("\n")))`

Reply:
👍 Didn't notice that kubectl-style format